### DISCUSSION TITLE
Windows 10 start menu doesn't display shortcut icons

### DISCUSSION THREAD

**Discussion by stamminator** [valinet/ExplorerPatcher#3460] [General]
I have a custom shortcut in `%APPDATA%\Microsoft\Windows\Start Menu\Programs` which launches Firefox in a specific profile. I changed the icon on this shortcut to differentiate them. This icon is shown in the Windows 10 start menu's app list, but the shortcut's pin shows the actual .exe file's icon rather than the shortcut's.

![ExplorerPatcher Windows 10 start menu shortcut icons bug](https://github.com/valinet/ExplorerPatcher/assets/10633170/e201a483-16f8-4f46-923f-eebbb9845913)

ExplorerPatcher version 22621.3810.66.1
Windows 11 build 22631.3737

**Comment by stamminator** [comment]
Is anyone able to reproduce this?

**Comment by pyrates999** [comment]
Try unpinning it and repinning the application.

Watch this to understand how pinning an application works in windows 11: https://www.youtube.com/watch?v=F3MMesBqlF0

**Reply by stamminator** [reply]
No difference, still shows the firefox.exe icon

**Reply by pyrates999** [reply]
Does it work if you uninstall EP?

**Reply by stamminator** [reply]
Yes, in the Windows 11 start menu, it honors the shortcut's icon

**Reply by pyrates999** [reply]
Please install the July windows update and see if that fixes it.

**Reply by stamminator** [reply]
Build 22631.3880, no difference. I've also reproduced on a new machine/install. Does the issue not occur for you?

**Reply by pyrates999** [reply]
I run windows 10, but with open shell installed so I can use the windows 7 start menu.  So I don't use pinning of apps in the start menu.

**Comment by stamminator** [comment]
Can we convert this into an issue?
